### PR TITLE
[GH-#369] Represent naming state using PMT

### LINKED ISSUE #369: Represent naming state using patricia merkle tree

### PR DESCRIPTION
**Pull request by DanielaIvanova**
resolves #369

### CODE REVIEW

**Comment by meivantodorov** [comment]
> I think it would be better to have the state transform for persistence, in the persistence module, than in the chain worker.

I think that persistence module should do exactly what the name says - `persist`.

**Comment by thepiwo** [comment]
I assume the transformation is only done for persistence, so this has nothing to do with the chain either.

File: apps/aeutil/lib/serialization.ex [hunk 1]
```diff
@@ -721,7 +721,7 @@ defmodule Aeutil.Serialization do
   def tag_to_type(24), do: OracleResponseTx
   def tag_to_type(25), do: OracleExtendTx
   def tag_to_type(30), do: Name
-  def tag_to_type(31), do: NameCommitmentTx
+  def tag_to_type(31), do: NameCommitment
```
Comment (thepiwo) [inline apps/aeutil/lib/serialization.ex]: is this changed on purpose?

Comment (DanielaIvanova) [reply apps/aeutil/lib/serialization.ex]: Yes, because was not working. Have to be like now

**Pull request merged**